velox/type/Variant.cpp

```diff
@@ -187,20 +187,25 @@ std::string variant::toJson(const TypePtr& type) const {
     return '"' + DATE()->toString(value<TypeKind::INTEGER>()) + '"';
   }
 
+  VELOX_CHECK(type);
+
+  VELOX_CHECK_EQ(this->kind(), type->kind(), "Wrong type in variant::toJson");
+
   switch (kind_) {
     case TypeKind::MAP: {
       auto& map = value<TypeKind::MAP>();
       std::string b{};
       b += "[";
       bool first = true;
+      VELOX_CHECK_GT(map.size(), 0, "No map elements available");
```

Comment:
This can be removed. We should verify this during map construction.

Reply:
Removed. The tests also removed for now.